projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -121,7 +118,16 @@ export class PdfViewerComponent implements AfterContentInit, OnChanges, OnDestro
       this.toolbarEvents.changePageByDeltaSubject.subscribe(pageNumber => this.pdfWrapper.changePageNumber(pageNumber))
     );
     this.$subscription.add(this.toolbarEvents.grabNDrag.subscribe(grabNDrag => this.enableGrabNDrag = grabNDrag));
-    this.$subscription.add(this.toolbarEvents.commentsPanelVisible.subscribe(toggle => this.showCommentsPanel = toggle));
+    this.$subscription.add(this.toolbarEvents.commentsPanelVisible.subscribe(toggle => {
+        this.showCommentsPanel = toggle;
+
+        if (toggle) {
```

Comment:
would setting the property in the html like below not work?
[scrollTop]="viewerContainer.scrollTop"